OpenRA.Mods.Common/Traits/Attack/AttackFrontal.cs

```diff
@@ -21,6 +21,9 @@ public class AttackFrontalInfo : AttackBaseInfo, Requires<IFacingInfo>
 		[Desc("Tolerance for attack angle. Range [0, 512], 512 covers 360 degrees.")]
 		public readonly new WAngle FacingTolerance = WAngle.Zero;
 
+		[Desc("The angle relative to the actor's orientation used to fire the weapon from.")]
+		public readonly WAngle[] FacingOffsets = { WAngle.Zero };
```

Comment:
I feel like this shouldn't be set in AttackFrontal, but rather in Armament. One may want alternating weapons, or different weapons having the ability to fire only in certain directions. Imagine a medieval ships, although it has cannons on both sides, it wants to only fire from the side which is pointing at the target